CHANGES.md

```diff
@@ -1,3 +1,17 @@
+3.15.1 (2024-04-17)
+-------------------
+
```

Comment:
I would add a "Fixed" subheader for this. Though looking at the older entries in the changelog, it's not really all that consistent.

Reply:
good idea!